src/NzbDrone.Core/NetImport/NetImportSearchService.cs

```diff
@@ -74,15 +78,76 @@ public List<Movie> MovieListSearch(int listId, bool onlyEnableAuto = false)
 
         public void Execute(NetImportSyncCommand message)
         {
-            var movies = FetchAndFilter(0, true);
+            var listedMovies = Fetch(0, true);
+            if (_configService.ListSyncLevel != "disabled")
+            {
+                var moviesInLibrary = _movieService.GetAllMovies();
+                foreach (var movie in moviesInLibrary)
+                    {
+                    bool foundMatch = false;
+                    foreach (var listedMovie in listedMovies)
+                    {
+                        if (movie.ImdbId == listedMovie.ImdbId)
+                        {
+                            foundMatch = true;
+                            break;
+                        }
+
+                    }
+                    if (!foundMatch)
+                    {
+                        if (_configService.ListSyncLevel == "logOnly")
```

Comment:
this has been changed in the latest commit: https://github.com/Radarr/Radarr/pull/656/commits/22f333313b3d17123743468591033b35eba532b6